x-men/frontend/public/template/menu.html

```diff
@@ -7,17 +7,34 @@
     <script src="{{scriptUri}}" defer></script>
 </head>
 <body>
-    <h1>SARA</h1>
-    <div class="options-wrapper">
-        <div>
-            <label for="location">Location:</label>
-            <select name="location" id="location">
-                <option value="local" selected="selected">Local</option>
-                <option value="remote">Remote</option>
-            </select>
-        </div>
+    <img src="{{imageUri}}sara.png" alt="SARA logo">
+    <div class="wrapper">
+        <div class="options-wrapper">
+            <div class="option-wrapper">
+                <div class="info-label-wrapper">
+                    <div class="tooltip"> &#9432;
+                        <span class="tooltiptext">
+                            <p>Selects which repository will be analyzed.</p>
+                            <ul>
+                                <li>
+                                    <p><b>Local</b> will analyze the local files.</p>
+                                </li>
+                                <li>
+                                    <p><b>Remote</b> will analyze the remote (GitHub) repository.</p>
+                                </li>
+                            </ul>
+                        </span>
+                    </div>
+                    <label for="location">Location:</label>
+                </div>
+                <div>
+                    <select name="location" id="location">
+                        <option value="local" selected="selected">Local</option>
+                        <option value="remote">Remote</option>
+                    </select>
+                </div>
+            </div>
+        <button id="submit">Submit</button>
```

Comment:
<button> is missing recommended "type" attributehtml-validate(no-implicit-button-type)
```suggestion
        <button id="submit" type="submit">Submit</button>
```